### PR TITLE
HCA add scpred, garnett and celltypes tool sections

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds sections for Cell typing tools for the HCA flavouring.

### CODE REVIEW

File: templates/galaxy/config/tool_conf.xml.j2 [hunk 1]
```diff
@@ -275,6 +275,12 @@
   </section>	
   <section id="hca_sc_scmap_tools" name="SCMap">
   </section>	
+  <section id="hca_sc_scpred_tools" name="SCPred">
```
Comment (bgruening) [inline templates/galaxy/config/tool_conf.xml.j2:278]: you can be more descriptive in the naming ...

Comment (pcm32) [reply templates/galaxy/config/tool_conf.xml.j2:278]: Will ask the person involved in these tools, thanks for taking a look.

**Comment by a-solovyev12** [comment]
@bgruening @pcm32 since all these tools are involved in cell type classification and description needs to be short, could you add something like: `ScPred - methods for scRNAseq cell type classification` and so on? Thanks!

**Comment by bgruening** [comment]
What about  "ScPred - scRNAseq cell type classification"

**Comment by a-solovyev12** [comment]
Sure, that should do as well

File: templates/galaxy/config/tool_conf.xml.j2 (hunk 1 above)
Comment (pcm32) [reply templates/galaxy/config/tool_conf.xml.j2:278]:
```suggestion
  <section id="hca_sc_scpred_tools" name="ScPred - scRNAseq cell type classification">
```

**Comment by pcm32** [comment]
Should we add more description to the few other tools @bgruening? Thanks

**Comment by bgruening** [comment]
Yes, think so.

**Comment by bgruening** [comment]
Is that still WIP?

**Comment by pcm32** [comment]
We are done, feel free to merge @bgruening

**Pull request merged**